.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "tests/metadata"]
 	path = modules/tests/metadata
-	url = https://github.com/coursier/test-metadata.git
+	url = https://github.com/LaurenceWarne/test-metadata.git
+	branch = update-for-scala3-tests
```

Comment:
Temporary until https://github.com/coursier/test-metadata/pull/6, hopefully tests should go green with it